create_teacher.rb

```diff
@@ -0,0 +1,18 @@
+require './teacher'
+require './person'
+require './nameable'
+
+def create_a_teacher
+  print 'Age: '
+  age = gets.chomp
+
+  print 'Name: '
+  name = gets.chomp
+
+  print 'Specialization: '
+  specialization = gets.chomp
+
+  @persons.push(Teacher.new(specialization, age, name))
+  puts 'Teacher created successfully!'
+  puts ''
+end
```

Comment:
- Great job addressing the issue pointed out by the previous reviewer, however, I can't create a teacher successfully. Please, kindly fix this issue of the argument below:


![Screenshot from 2023-01-19 08-45-56](https://user-images.githubusercontent.com/59397159/213385671-d635850a-eda4-467c-9617-f26b941e588b.png)